### PR TITLE
nats MitM vulnerability

### PR DESCRIPTION
**Pull request by paolobarbolini**
copy-pasted from #1661.
~~No fixes have been released since the crate is considered deprecated (see https://github.com/nats-io/nats.rs#nats)~~

### CODE REVIEW

File: nats/RUSTSEC-0000-0000.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+keywords = ["tls", "mitm"]
+
+[versions]
+unaffected = ["< 0.9.0"]
+```
+
+# TLS certificate common name validation bypass
+
+The NATS official Rust clients are vulnerable to MitM when using TLS.
+
```
[the remark this answers is not in the document]
Comment (paolobarbolini) [reply nats/RUSTSEC-0000-0000.md:16]: Actually they told me there will be a fix but after the current rustls alpha is released as stable. Sorry @pinkforest  for all of the time you spent editing the message.

[the remark this answers is not in the document]
Comment (pinkforest) [reply nats/RUSTSEC-0000-0000.md:16]: No worries - Could we put some wording around that and include async-nats as a actionable option ?

**Comment by pinkforest** [comment]
Thanks :partying_face:

**Comment by pinkforest** [comment]
This was in the wrong directory....

**Comment by paolobarbolini** [comment]
Ops 😳

**Comment by pinkforest** [comment]
I should have caught it - Fixed :) :partying_face:

**Pull request merged**